### PR TITLE
feat: LT-956 - Advice view on case

### PR DESCRIPTION
**Pull request by cstobart**

### CODE REVIEW

File: cases/urls.py [hunk 1]
```diff
@@ -24,4 +24,6 @@
     path('<uuid:pk>/documents/<str:file_pk>/', views.Document.as_view(), name='document'),
     # ex: /cases/<uuid:pk>/assign-flags/
     path('<uuid:pk>/assign-flags/', views.AssignFlags.as_view(), name='assign_flags'),
+
```
Comment (jfaracik) [inline cases/urls.py]: Add # example

File: ui_automation_tests/step_defs/test_manage_cases.py [hunk 2]
```diff
@@ -156,3 +156,9 @@ def i_see_ultimate_end_user_on_page(driver, context):
         assert context.ueu_website in destinations_table
         assert context.ueu_address in destinations_table
         assert context.ueu_country[0] in destinations_table
+
+
+    @when('I click on advice view')
+    def i_click_on_advice_view(driver, context):
```
Comment (markm5A8Z9F43913A) [inline ui_automation_tests/step_defs/test_manage_cases.py]: Remove one emplty line before method. Add one to the end of file.

File: ui_automation_tests/features/manage_cases.feature [hunk 3]
```diff
@@ -100,4 +100,10 @@ Feature: I want to record the final decision overall on an application case
...
     When I click on application previously created with pre incorporated goods
-    Then I see an ultimate end user
\ No newline at end of file
+    Then I see an ultimate end user
+
+  @LT-956_can_see_advice_view
+  Scenario: Gov user can see advice view page
+    Given I go to internal homepage
+    When I click on application previously created
+    When I click on advice view
```
Comment (markm5A8Z9F43913A) [inline ui_automation_tests/features/manage_cases.feature]: Given, When, And.

And you might also want to add a Then, to make an assertion there.

Comment (cstobart) [reply ui_automation_tests/features/manage_cases.feature]: The 'Thens' will come when we play the next stories

File: ui_automation_tests/pages/application_page.py [hunk 4]
```diff
@@ -81,7 +82,12 @@ def get_text_of_activity_users(self, no):
     def click_edit_case_flags(self):
         edit_cases_btn = self.driver.find_element_by_id(self.edit_case_flags)
         edit_cases_btn.click()
-        
+
+
+    def click_view_adivce(self):
```
Comment (markm5A8Z9F43913A) [inline ui_automation_tests/pages/application_page.py]: Typo

Comment (markm5A8Z9F43913A) [reply ui_automation_tests/pages/application_page.py]: Also too many empty lines around method

**Pull request merged**